Overview update (#3605)
Nexmo/nexmo-developer: _documentation/en/client-sdk/in-app-voice/overview.md

---
title: Overview
description: This topic provides an overview of how to build in-app voice applications using the Client SDK.
navigation_weight: 0
---

# In-App Voice Overview

> In-App voice enabled via the Client SDK enables IP based voice communication directly within your web (Javascript), Android, and iOS applications.  With the Client SDK you can easily embed voice into your contact center, marketplace and B2C applications supporting: app to app calling, click-to-call, conferencing, Voice API features such as Automatic Speech Recognition, Websockets, TTS and more. Using the Client SDK also allowes you to easily interconnect In-App voice and In-App messaging.

Reviewer: ```suggestion
The Client SDK makes it straightforward to include IP-based voice communication capabilities in your web (JavaScript), Android, and iOS applications. Use voice within your apps to build [contact center](/use-cases/contact-center-client-sdk), [marketplace](/use-cases/digital-marketplace-client-sdk) and B2C (Business to Consumer) solutions that include:

* [App to app calling](/client-sdk/tutorials/app-to-app/introduction/)
* [Click to call](/use-cases/client-sdk-click-to-call)
* [Audio conferencing](/voice/voice-api/code-snippets/connect-callers-into-a-conference)
* [Automatic speech recognition](/voice/voice-api/guides/asr)
* [Websockets](/voice/voice-api/guides/websockets)
* [Text-to-speech](/voice/voice-api/guides/text-to-speech)

The Client SDK also enables you to include [in-app voice](/client-sdk/in-app-voice/overview) and [in-app messaging](/client-sdk/in-app-messaging/overview) in your applications so that your customers can communicate with you using their preferred channels.
```

### Features Include

> Client SDK In-App Voice uses WebRTC and includes all the essentials you need to build a feature-rich voice experience

Reviewer: ```suggestion
Client SDK In-App Voice uses WebRTC and includes all the essentials you need to build a feature-rich voice experience that includes:
```

* 1:1 or Group Calls
* Audio Controls – Mute, earmuff
* DTMF Support

### Extendability Through the Voice API

> Client SDK In-App Voice is a part of the Vonage Voice API which amplifies the In-App Voice offering through extra functionality

Reviewer: ```suggestion
Client SDK In-App Voice is a part of the Vonage Voice API which amplifies the In-App Voice offering by enabling:
```

* Conference calling
* Text-to-speech messages in 23 languages

### Native SDK Specific Features

> The Android and iOS Client SDKs offer additional capabilities

Reviewer: ```suggestion
The Android and iOS Client SDKs offer additional capabilities, including:
```

* Network Change Handling
* Audio Routing Management
* Push Notifications

## Voice calls

> **The way to make a voice call is using the Client SDK `callServer()` method. This allowes for the use of the [Voice API](/voice/voice-api/overview) to manage the call on the server side. The Client SDK application manages the [Event flow](/conversation/guides/event-flow) of the conversation in the client side.**

Reviewer: ```suggestion
**To make a voice call, use the Client SDK `callServer()` method. This enables you to use the [Voice API](/voice/voice-api/overview) to manage the call on the server. The Client SDK application manages the [event flow](/conversation/guides/event-flow) of the conversation in the client.**
```

## Reference

* [Client SDK Reference - Web](/sdk/client-sdk/javascript)
* [Client SDK Reference - iOS](/sdk/client-sdk/ios)
* [Client SDK Reference - Android](/sdk/client-sdk/android)

> For advanced control of voice calls and the converations containing them see [conversation](/conversation/overview)

Reviewer: ```suggestion
For more information about managing conversations that include voice calls, see the [Conversation API](/conversation/overview)
```
